hello-json.roc

```diff
@@ -0,0 +1,38 @@
+app "example"
```

Comment:
Should both hello-json.roc files be kept? There's currently also one in the applications folder.